src/components/ColourBar.vue

```diff
@@ -4,6 +4,28 @@
                                 $vuetify.breakpoint.mobile ? 'colourbar-mobile' : 'colourbar-desktop',
                                 $vuetify.theme.dark ? 'colourbar-dark' : 'colourbar-light'
                                 ]"/>
+    <v-text-field ref="minInput" v-if="isEditingMin"
+      @keydown.enter.stop="updatedRange"
+      @keydown.escape.stop="updatedRange"
```

Comment:
The escape button now doesn't revert back to the old value, but accepts the input. This is a bit counterintuitive